crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -197,6 +233,9 @@ impl Node for BloomNode {
         self.view_query.update_archetypes(world);
     }
 
+    // Atypically for a post-processing effect, we do not
```

Comment:
```suggestion
    // Atypically for a post-processing effect, we do not need to
```

Reply:
I'll commit this but may i ask why? Isn't this no more than a stylistic change that actually makes the comment a little less clear? ("we don't need to use" does't imply that we don't actually use it, while "we don't use" definitely does)

Reply:
Ah I see your point. I was trying to clarify that it's not like we _couldn't_ use the double buffer, but that we _didn't_ need to. Feel free to change the wording if you think of something better.